docs/data/material/customization/theme-components/theme-components.md

```diff
@@ -100,7 +100,9 @@ The `sx` prop acts as a shortcut for defining custom styles that access the them
 This prop lets you write inline styles using a superset of CSS.
 Learn more about [the concept behind the `sx` prop](/system/the-sx-prop/) and [how `sx` differs from the `styled` utility](/system/styled/#difference-with-the-sx-prop).
 
-You can use the `sx` prop inside the `styleOverrides` key to modify styles within the theme using shorthand CSS notation. This is especially handy if you're already used to using the `sx` prop syntax. It makes transferring one-off customizations using `sx` to the theme very fast.
+You can use the `sx` prop inside the `styleOverrides` key to modify styles within the theme using shorthand CSS notation.
+This is especially handy if you're already used to using the `sx` prop syntax.
+It makes transferring one-off customizations using `sx` to the theme very fast.
```

Comment:
@samuelsycamore am I being too repetitive in mentioning `sx` three times in the same paragraph? 😅

Reply:
I think it's always preferable to err on the side of accuracy if using pronouns would make things unclear.

If you wanted to rephrase it (totally optional), you could link the ideas in the last two sentences like this:

```suggestion
You can use the `sx` prop inside the `styleOverrides` key to modify styles within the theme using shorthand CSS notation.
This is especially handy if you're already using the `sx` prop with your components, because you can use the same syntax in your theme and quickly transfer styles between the two.
```